package.json

```diff
@@ -19,7 +19,14 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "release": "yarn build && auto shipit",
-    "chromatic": "npx chromatic --project-token=b80523143ab2"
+    "chromatic": "npx chromatic --project-token=b80523143ab2",
```

Comment:
Should we move the project token from package.json to an `.env` `CHROMATIC_PROJECT_TOKEN` variable, in keeping with [this recent PR](https://github.com/chromaui/chromatic-docs/pull/223) to the docs?

Reply:
yea, we already use that with the GH action. I'll remove this script.